engine/src/main/java/org/terasology/rendering/opengl/FBO.java

```diff
@@ -104,37 +104,39 @@ public void bind() {
      * Meanwhile shaders might output not just colors but additional per-pixel data. This method establishes on which
      * of an FBOs attachments, subsequent opengl commands and shaders will draw on.
      *
-     * @param c If True the color buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
-     * @param n If True the normal buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
-     * @param l If True the light buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
+     * @param color If True the color buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
+     * @param normals If True the normal buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
+     * @param lightBuffer If True the light buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
      */
-    public void setRenderBufferMask(boolean c, boolean n, boolean l) {
-        if (this.color == c && this.normal == n && this.lightBuffer == l) {
+    public void setRenderBufferMask(boolean color, boolean normals, boolean lightBuffer) {
```

Comment:
I feel the variables in this whole method are somewhat misleading. In the physical world if something is "masked" (out) it shouldn't receive any ink or paint. So if "color" is "true" nothing should get written on the color buffer as it is masked out. This method however uses the term "mask" somewhat ambiguously and reverse its meaning to "masked-in" so that if "color" is true shaders can write to the color buffer instead. But I don't think "masked-in" is the commonly thought meaning of the term "mask".

So, how about we change all the relevant variables and method arguments to:

- writeToColorBuffer
- writeToNormalsBuffer
- writeToLightBuffer

so that what's happening is clearer and less ambiguous?